uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/wizard/JMXNewConnectionWizard.java

```diff
@@ -0,0 +1,183 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+/** 
+ * This class 'FuseJMXNewConnectionWizard.java' was generated by RedDeer Code Generator. 
+ * Selected options:
+ *		Getter
+ *		Setter
+ *		Action method
+ *		Generate static constants
+ *		Include all wizard pages
+ *		Allow inheriting (only, if are included all wizard pages)
+ */
+package org.jboss.tools.fuse.reddeer.wizard;
+
+import org.eclipse.reddeer.jface.wizard.WizardDialog;
+import org.eclipse.reddeer.swt.impl.button.CheckBox;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.button.RadioButton;
+import org.eclipse.reddeer.swt.impl.shell.DefaultShell;
+import org.eclipse.reddeer.swt.impl.text.LabeledText;
+import org.eclipse.reddeer.swt.impl.tree.DefaultTree;
+import org.eclipse.reddeer.swt.impl.tree.DefaultTreeItem;
+
+/**
+ * Represents <i>JMX New Connection</i> dialog that is displayed after click on <i>New Connection...</i> button in
+ * <i>JMX Navigator</i> view.
+ * 
+ * @author djelinek
+ */
+public class JMXNewConnectionWizard extends WizardDialog {
```

Comment:
Please distinguish between WizardDIalog and WizardPage. We used to have everything in WizardDialog with RedDeer 1.x but now we should really start distinguishing between them.